packages/orbit-components/src/Itinerary/Itinerary.stories.tsx

```diff
@@ -385,163 +641,131 @@ export const ItinerarySeparatorComponent = () => {
-          <ItinerarySegmentStop
-            city="Brno BRQ"
-            station="Brno-Tuřany"
-            date="Mon, 30.1"
-            time="17:30"
-          />
-          <ItinerarySegmentDetail
-            duration="1h 35m"
-            summary={
-              <Badge carriers={[{ code: "FR", name: "Ryanair" }]} border={false}>
-                Ryanair
-              </Badge>
-            }
-            content={content}
-          />
-          <ItinerarySegmentStop
-            city="Barcelona BCN"
-            station="El Prat de Llobregat"
-            hidden
-            date="Mon, 30.1"
-            time="20:00"
-          />
-          <ItinerarySegmentStop
-            city={<Text type="secondary">New York JFK</Text>}
-            station={
-              <Stack flex align="center" spacing="200">
-                <CountryFlag code="US" size="small" />
-                <Text type="secondary" size="small">
-                  United states
-                </Text>
-              </Stack>
-            }
-          />
-        </ItinerarySegment>
-      </Itinerary>
-      <Heading type="title2">Throwaway ticketing</Heading>
-      <Itinerary>
-        <ItinerarySegment
-          banner={
-            <Stack inline align="stretch">
-              <ItinerarySegmentBanner>
-                <ItineraryBadgeList>
-                  <ItineraryBadgeListItem type="info" icon={<StarFull color="info" />}>
-                    <Text as="span" type="info" weight="bold">
-                      Throwaway ticketing hack:{" "}
-                    </Text>{" "}
-                    You are saving money with this travel hack.
-                  </ItineraryBadgeListItem>
-                </ItineraryBadgeList>
-              </ItinerarySegmentBanner>
-            </Stack>
-          }
-        >
-          <ItinerarySegmentStop
-            city="Barcelona BCN"
-            station="Barcelona-Sants"
-            date="Mon, 30.1"
-            time="17:30"
-          />
-          <ItinerarySegmentDetail
-            duration="2h 30m"
-            summary={
-              <Badge carriers={[{ code: "FR", name: "Ryanair" }]} border={false}>
-                Ryanair
-              </Badge>
-            }
-            content={content}
-          />
-          <ItinerarySegmentStop
-            city="London LHR"
-            station="London Heathrow"
-            date="Mon, 30.1"
-            time="20:00"
-          />
-        </ItinerarySegment>
-      </Itinerary>
-    </Stack>
-  );
-};
+  parameters: {
+    info: "This components shows the ItinerarySeparator component. For color prop, use a value defined Orbit theme colors. Check Orbit.Kiwi for more detailed design guidelines.",
+    controls: {
+      exclude: ["spaceAfter"],
+    },
+  },
 
-Stop.args = {
-  date: "Fr, 19.10",
-  time: "14:05",
-  station: "Václav Havel Airport Prague (PRG)",
-  city: "Prague",
-  type: undefined,
-  hidden: false,
-};
+  args: {
+    color: "paletteRedDarkHover",
```

Comment:
If I looked correctly, we have defined 77 variants of palette colors that could be used. It would be better to have it as a select list, to make it clearer for the user. I have to just think where to put the enum with defined colors, as I guess we don't wanna have ~80 additional lines just for an enum in `stories.tsx` file. 😄